google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2605,6 +2781,16 @@ enum Position {
   private long numLeakedSessionsRemoved = 0;
 
   private AtomicLong numWaiterTimeouts = new AtomicLong();
+  private AtomicLong numMultiplexedSessionWaiterTimeouts = new AtomicLong();
```

Comment:
Remove this. The multiplexed session is either:
1. Created and available.
2. Still being created. The 'waiter' should just wait for the configured timeout.
3. Failed. We should return an error.

Reply:
See my comments further above. I don't think this metric makes sense for multiplexed sessions. Threads won't be waiting for a multiplexed session in the same way as for regular sessions, meaning that this metric would only potentially be increased during startup (and then also only if the user has changed the default timeout configuration).

Reply:
Sure, removed the metric.